docs/development.md

```diff
@@ -403,6 +403,7 @@ You can optionally set the following variables:
 | `SKIP_CREATE_MGMT_CLUSTER` | Skip management cluster creation. If skipping managment cluster creation you must specify `KUBECONFIG` and `SKIP_CLEANUP` | `false`                                                                               |
 | `LOCAL_ONLY`               | Use Kind local registry and run the subset of tests which don't require a remotely pushed controller image.               | `true`                                                                                |
 | `REGISTRY`                 | Registry to push the controller image.                                                                                    | `capzci.azurecr.io/ci-e2e`                                                            |
+| `CLUSTER_NAME`              | Name of an exisiting workload cluster.  Will run specs against existing workload cluster. Use in conjunction with `SKIP_CREATE_MGMT_CLUSTER` and `KUBECONFIG` | `capz-e2e-uf9yll`                                       |
```

Comment:
one test run != one workload cluster, this might lead to unexpected results if not used with GINKGO_FOCUS (assuming it will re-apply the template for each spec using the same cluster names, especially if running the different specs in parallel)

Reply:
Yes. I will update the docs to point that out.

Reply:
updated to clarify